SEED-platform/seed
# refactor: only import AT scenarios with measures or meters

#### Any background context you want to provide?

#### What's this PR do?
- tweaks the import of scenarios from Audit Template so that we skip scenarios that don't have meters or measures. Note that this does _not_ apply to non-AT files!
- start importing TemporalStatus from BuildingSync scenarios

#### How should this be manually tested?
Import the example SF AT file here: https://buildingenergyscore.energy.gov/documents/example_SF_audit_report_BS_v2.3_081321.xml

You should only see it import the 4 "valid" scenarios which are:
- lighting package of measures
- HVAC package of measures
- electricity meter data
- fuel oil "meter" data


#### What are the relevant tickets?
#2961 
Closes #2830 

#### Screenshots (if appropriate)
<img width="1377" alt="Screen Shot 2021-10-20 at 2 13 20 PM" src="https://user-images.githubusercontent.com/18518728/138166488-f2ce7a7c-0c78-4c73-9781-806e6dda998f.png">
<img width="722" alt="Screen Shot 2021-10-20 at 2 13 44 PM" src="https://user-images.githubusercontent.com/18518728/138166543-96f66c7b-7b52-4b22-b6b1-29dbb15f5bd6.png">
